Website/structure-sources/index.rakudoc

```diff
@@ -7,10 +7,10 @@
         <div class="hero-body">
           <div class="container">
             <h1 class="title is-1 is-size-2-mobile has-text-centered">
-              Raku™ documentation
+              Raku documentation
```

Comment:
I couldn't keep up - do we want to keep the ™ here or not? Anyway, probably it shouldn't change in one monolithic PR about other changes.

Reply:
@coke removed many TM. This is correct - I think.

Reply:
No, this was intentionally added. same with the other one.